### DISCUSSION TITLE
功能&优化点征集

### DISCUSSION THREAD

**Discussion by anobaka** [anobaka/Bakabase#268] [Ideas]
欢迎大家直接在本issue下提出想要的新功能和优化点~
会按照需求优先级开发~

**Comment by Tandio** [comment]
首先感谢大佬做的软件，之前找了好久没有称心的资源管理软件。赞美大佬。
然后有些小的优化建议：
1. 希望能够实现自定义的资源名称解析功能。
2. 资源管理界面在进行批量操作时缺少全选的功能，需要一个个去点击不太方便，希望增加全选按钮，如果能做得更好的话，希望能够实现shift+鼠标左键全选的功能(类似windows自带的shift+鼠标进行多选的功能)
3. 资源管理界面的查询功能能够增加模糊查询(例如想搜索作者 + 年份的组合条件时，可以输入"作者 年份"进行查询)或者正则查询。目的是为了便于根据资源文件名称查询后进行批量的自定义标签

**Comment by anobaka** [comment]
感谢你的使用~

> 希望能够实现自定义的资源名称解析功能。

这个可以麻烦给个想要的形式吗

**Comment by Tandio** [comment]
> 这个可以麻烦给个想要的形式吗

类似于calibre的正则识别([地址](https://blog.bgme.me/posts/use-calibre-to-manage-your-doujinshi-2/))：
![image](https://user-images.githubusercontent.com/49389044/187212422-5a0bb6f2-a770-4957-8b5c-457d7f9b43ee.png)

**Reply by anobaka** [reply]
#240

**Comment by liuhanqing-phy** [comment]
大佬可以优先做一下[bangumi增强器](https://github.com/anobaka/InsideWorld/issues/246)吗，因为我主要用您的软件管理动画资源，如果能拉取bangumi的信息那再好不过了。
有个jellyfin从bangumi拉取番剧信息的项目，我觉得您可以参考一下
[[bgm.tv](https://bgm.tv/) metadata provider for Jellyfin](https://github.com/kookxiang/jellyfin-plugin-bangumi)

**Reply by anobaka** [reply]
收到，肝完1.7.0就开始

**Reply by liuhanqing-phy** [reply]
十分感谢！

**Comment by anobaka** [comment]
#332

**Comment by anobaka** [comment]
支持 ~~Linux、~~  MacOS

**Comment by anobaka** [comment]
转移至 https://github.com/anobaka/InsideWorld/discussions/506
